tls/s2n_tls.h

```diff
@@ -27,9 +27,10 @@ int s2n_flush(struct s2n_connection *conn, s2n_blocked_status *more);
 S2N_RESULT s2n_client_hello_request_validate(struct s2n_connection *conn);
 S2N_RESULT s2n_client_hello_request_recv(struct s2n_connection *conn);
 int s2n_client_hello_send(struct s2n_connection *conn);
+int s2n_parse_client_hello(struct s2n_connection *conn);
```

Comment:
If you're going to make this method "public" than you should probably give it a better name: s2n_client_hello_parse.

Reply:
Am I missing something 🤔 ? Why do you need to make this method public at all?